web/styles/popovers.css

```diff
@@ -615,16 +662,67 @@ ul.popover-group-menu-member-list {
         }
     }
 
-    .subscription-group-list,
-    .subscription-stream-list {
+    .stream-list-container {
+        height: 40vh;
+        overflow: hidden;
+        border-radius: 4px;
+        background-color: inherit;
+        border: 1px solid var(--color-border-user-profile-table);
+        margin-bottom: 5px;
+        -webkit-overflow-scrolling: touch;
+
+        .user-profile-channel-row {
+            display: flex;
+            justify-content: space-between;
+            padding: 5px 10px;
+            align-items: center;
+
+            .user-stream-list-item {
+                display: flex;
+                overflow: hidden;
+            }
+
+            .user-group-name,
+            .user-stream-name {
+                white-space: nowrap;
+                overflow: hidden;
+                text-overflow: ellipsis;
+                padding-right: 2px;
+            }
+        }
+
+        .subscription-stream-list {
+            max-height: calc(40vh - 2.4em);
```

Comment:
How did you get this 2.4em? Height of search input is 1.4em. This is how it looks at 12px font size - 
<img width="669" height="341" alt="image" src="https://github.com/user-attachments/assets/8cfdd6bd-d3a3-4bda-8780-e81f3a3c35d2" />